### PR TITLE
Preload Steel Connections for Dynamo package from Revit

### PR DESCRIPTION
**Pull request by vpavel04**
Preload Steel Connections for Dynamo package from Revit

### CODE REVIEW

**Comment by vpavel04** [comment]
I believe that Dynamo4Revit should have an Addin or an Extension mechanism to preload additional nodes and to extend LayoutSpecs.json properly. For the moment, I believe this is the best solution until Dynamo4Revit will be able to dynamically discover nodes to preload

File: src/DynamoRevit/RevitPathResolver.cs [hunk 1]
```diff
@@ -55,6 +55,19 @@ internal RevitPathResolver(string userDataFolder, string commonDataFolder)
             // Add the Revit_20xx folder for assembly resolution
             additionalResolutionPaths = new List<string> { currentAssemblyDir };
 
+			// Add Steel nodes to preload
+			var steelNodesDirectory = Path.Combine(currentAssemblyDir, "nodesSteelConnections", "steel-pkg", "bin");
```
Comment (QilongTang) [inline src/DynamoRevit/RevitPathResolver.cs]: Is the format here broken or it's just a github display issue?

Comment (vpavel04) [reply src/DynamoRevit/RevitPathResolver.cs]: i've had tabs instead of spaces. I've fixed it

**Comment by QilongTang** [comment]
@AndyDu1985 LGTM, what do you think?

**Comment by vpavel04** [comment]
i will come back with the generic mechanism

**Pull request closed without merging**